### DISCUSSION TITLE
Ignoring the schedule from the dashboard

### DISCUSSION THREAD

**Discussion by northern-mindset** [renovatebot/renovate#10509] [Request Help] [answered]
In the “Awaiting Schedule” section of the dashboard the sub-title says:

> These updates are awaiting their schedule. Click on a checkbox to ignore the schedule.

What does “ignore” actually mean? Probable options are:

1. To proceed with the update immediately (not waiting for the schedule).
2. Not to proceed with the update at the scheduled time (effectively to skip the update).

Of course I guess it's the first option, but the wording is not that clear.

**Comment by HonkingGoose** [comment, marked as answer]
When you click the checkbox Renovate bot creates a pull request to update that dependency now, without waiting for the scheduled time slot. This is option 1 from your list. 😉 

I agree the wording can be confusing if you don't know the behavior of the bot already...

@rarkins Do we want to change the wording to something more specific? Like:

- Click on a checkbox to force a update now
- Click on a checkbox to update now
- Click on a checkbox to get a update now
- Click on a checkbox to ignore the schedule and get a update now

**Reply by rarkins** [reply]
Let's make it clear that checking a box means "open PR now"

**Reply by HonkingGoose** [reply]
I'll try to make the change then!

**Reply by northern-mindset** [reply]
Sometimes Renovate does not create a PR right away even if schedule is “any time” (as far as I know). I mean the [`prCreation`](https://docs.renovatebot.com/configuration-options/#prcreation) option. Therefore I'd suggest something more generic that I tried to express as “proceed with the update” in the question.

**Reply by HonkingGoose** [reply]
I've opened a PR (#10512).
